manifests/h/Helios/TextPad/9.2.0/Helios.TextPad.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Helios.TextPad
+PackageVersion: 9.2.0
+Installers:
+- InstallerUrl: https://www.textpad.com/file?path=v9/setupv9.exe
+  Architecture: x64
+  InstallerType: exe
+  InstallerSha256: C43E7F36DB9C43F6856B7C651C2BC907899FA7962DA8AFEA6F8EA4149D2874B1

```

Comment:
```suggestion
  InstallerSha256: 862441694DFC3DEE327103CF56FC3079924B0700750070E8EEB2DFBECA440C25
```

Reply:
This sha256 above mismatch with my computer.  .2420BD2DFE2751E4F8DFFE6875D6026B62F6A90CEB142803A98323C2A6770631